synapse/common.py

```diff
@@ -157,13 +157,39 @@ def intify(x):
     except (TypeError, ValueError):
         return None
 
-hugectx = decimal.Context(prec=15)
+hugectxv1 = decimal.Context(prec=15)
+def hugenumv1(valu):
+    '''
+    Return a decimal.Decimal with original hugenum precision.
+    '''
+    return decimal.Decimal(valu, context=hugectxv1)
```

Comment:
Should this function be marked as deprecated so we know its a candidate for removal in the future?

Reply:
Yeah i'll mark it